.github/workflows/zz_generated.create_release_pr.yaml

```diff
@@ -65,7 +67,7 @@ jobs:
       DIR: "/opt/cache"
       IMAGE: "quay.io/giantswarm/architect"
       IMAGE_PATH: "/usr/bin/architect"
-      VERSION: "2.1.4"
+      VERSION: "1.2.0"
```

Comment:
There is a fix for the bug updating go.mod in architect 3.0.0 but using the latest devctl didn't update the version. Will check on that later.